app/renderer/js/components/handle-external-link.ts

```diff
@@ -9,7 +9,7 @@ const { shell, app } = remote;
 const dingSound = new Audio('../resources/sounds/ding.ogg');
 
 // TODO: TypeScript - Figure out a way to pass correct type here.
-function handleExternalLink(this: any, event: any): void {
+function handleExternalLink(this: any, event: Electron.NewWindowEvent): void {
```

Comment:
Should be `this: WebView` (with `import type WebView from './webview'`).

Reply:
Cannot find namespace 'WebView'   .ts(2503)

Reply:
It works fine on `master`. Did you forget to run `npm install` to follow the upgrades we did to TypeScript and other dependencies?

Reply:
solved